pkg/smartcontract/rpcbinding/binding.go

```diff
@@ -208,6 +210,7 @@ func NewReader(invoker Invoker{{- if not (len .Hash) -}}, hash util.Uint160{{- e
 		{{- if .IsNep11D}}*nep11.NewDivisibleReader(invoker, hash), {{end}}
 		{{- if .IsNep11ND}}*nep11.NewNonDivisibleReader(invoker, hash), {{end}}
 		{{- if .IsNep17}}*nep17.NewReader(invoker, hash), {{end -}}
+		{{- if .IsNep24}}nep24.NewRoyaltyReader(invoker, hash), {{end -}}
```

Comment:
And don't forget to add dereference after `nep24.NewRoyaltyReader` return value change.